example/eg-flag-set/go.mod

```diff
@@ -0,0 +1,10 @@
+module eg-flag-set
+
+go 1.12
+
+replace github.com/jpillora/opts => ../../
```

Comment:
This go.mod file is needed so the entire library does not become dependant on glog,

Reply:
Oh? Even though https://github.com/jpillora/opts/blob/subcommands/go.mod doesn't have glog in it?

Reply:
Do a `go build` of `example/eg-flag-set` and it will pitch-up.

Reply:
Yep go.mod will trigger auto-get, though if we don't have this go.mod file, and users do a `go get github.com/jpillora/opts` will they get `glog`?

Reply:
The second go.mod file in the example means `go get github.com/jpillora/opts` with not get glog. A `go get github.com/jpillora/opts/...` would get it.